docs/spmd.md

```diff
@@ -130,7 +130,7 @@ In general, SPMD programs should create a single mesh and reuse it for all shard
 Mesh nicely abstracts how the physical device mesh is constructed. Users can arrange devices in any shape and order using the logical mesh. However, one can define a more performant mesh based on the physical topology, especially when it involves Data Center Network (DCN) cross slice connections. HybridMesh creates a mesh which gives good performance out of the box for such multislice environments. It accepts ici\_mesh\_shape and dcn\_mesh\_shape which denote logical mesh shapes of inner and outer network.
 
 ```python
-from torch_xla.experimental.xla_sharding import HybridMesh
```

Comment:
I think @will-cromar did something to aliasing the old experimental PJRT API with the new one(and throw a warning message) when moving pjrt out of experimental. I think we should do the same for SPMD, otherwise we break the backward compatibility

Reply:
Agreed, I will look into it and add some warning msgs to the `experimental.spmd/__init__.py` when we move out of experimental.

Reply:
```
>>> import torch_xla.experimental.xla_sharding as xs
<stdin>:1: DeprecationWarning: Importing from `torch_xla.experimental.xla_sharding` will be deprecated after 2.2 release. Please use `torch_xla.experimental.spmd` instead.
```

Reply:
do you want to just move it out of experimental all-togeather? I think we can call it beta in 2.2 release

Reply:
Ack, sounds good to me.

Reply:
Done.